arch/arm64/configs/sc7280_defconfig

```diff
@@ -348,6 +349,7 @@ CONFIG_DRM_MSM=y
 # CONFIG_DRM_MSM_DSI_10NM_PHY is not set
 # CONFIG_DRM_MSM_HDMI is not set
 CONFIG_DRM_PANEL_RAYDIUM_RM692E5=y
+CONFIG_DRM_PANEL_SHIFT_SH8804B=m
```

Comment:
you don't want to have those built-in? Also FOCALTECH_FT3658U? This defconfig is imo only for development anyways so making everything built-in simplifies development